content/tests.py

```diff
@@ -464,6 +465,21 @@ def test_end_of_event_task_event_with_answers_for_multiple_learners_first_awarde
 
         self.assertEquals(awarded_badge.count(), 0)
 
+    @patch("mobileu.tasks.mail_managers")
+    def test_sumit_counts(self, mocked_mail_managers):
+        s = SUMit.objects.create(name='Blarg',
+                                 course=self.course,
+                                 activation_date=datetime.now()+timedelta(hours=12),
+                                 deactivation_date=datetime.now()+timedelta(hours=24))
+        s.save()
```

Comment:
This is good, just thinking maybe you should also check that the counts returned by get_question_counts method are correct.